prog/vnet/update_firewall_rules.rb

```diff
@@ -23,8 +28,63 @@ class Prog::Vnet::UpdateFirewallRules < Prog::Base
     flags interval;
 #{allowed_ingress_ip6.any? ? "elements = {#{allowed_ingress_ip6.join(",")}}" : ""}
   }
+
+  set allowed_ipv4_port_tuple {
+    type ipv4_addr . inet_service;
+    flags interval;
+#{allowed_ingress_ip4_port_tuple.any? ? "elements = {#{allowed_ingress_ip4_port_tuple.map { |ip, port| "#{ip} . #{port}" }.join(",")}}" : ""}
+  }
+
+  set allowed_ipv6_port_tuple {
+    type ipv6_addr . inet_service;
+    flags interval;
+#{allowed_ingress_ip6_port_tuple.any? ? "elements = {#{allowed_ingress_ip6_port_tuple.map { |ip, port| "#{ip} . #{port}" }.join(",")}}" : ""}
+  }
+
+  set private_ipv4_ips {
+    type ipv4_addr;
+    flags interval;
+    elements = {
+      #{generate_private_ip4_list}
+    }
+  }
+
+  set private_ipv6_ips {
+    type ipv6_addr
+    flags interval
+    elements = { #{generate_private_ip6_list} }
+  }
+
+  chain forward_ingress {
+    type filter hook forward priority filter; policy drop;
+    tcp dport 22 ct state new,established,related accept
```

Comment:
This one should go, and github runners and or the pool should add a rule like every other civilian

Reply:
but this is for ssh access. Shouldn't all of the instances already allow dport 22? Currently wait_sshable would fail otherwise? Do you suggest we remove this and add it as a rule by default?

Reply:
Well, you're right that it would break wait sshable, which is a problem, but no, instances in general should not have port 22 exposed regardless of ruleset.

Reply:
Let's ship it for now, but if we have to eliminate wait_ssh from the obliged VM startup, let's do it.

Reply:
I have actually made this change. We're adding a default rule to allow dport 22. I think that's OK?